scripts/gen-device.sh

```diff
@@ -18,7 +18,7 @@ openssl pkcs8 -topk8 -nocrypt -in "${device_dir}/pkey.ec.pem" -out "${device_dir
 openssl req -new -key "${device_dir}/pkey.pem" \
           -config <(sed "s/\$ENV::DEVICE_UUID/${DEVICE_UUID}/g" "${CWD}/certs/client.cnf") \
           -out "${device_dir}/${device_id}.csr"
-  
+
```

Comment:
It's generally considered polite to put whitespace/formatting changes in their own PR.